tools/c7n_gcp/c7n_gcp/filters/essential_contacts.py

```diff
@@ -0,0 +1,74 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+"""
+Essential Contacts suppport for GCP organization
+"""
+from c7n.filters.core import ValueFilter
+from c7n.utils import local_session, type_schema
+from c7n_gcp.resources.resourcemanager import Organization
+import jmespath
+
+
+@Organization.filter_registry.register('essential-contacts')
+class EssentialContactsFilter(ValueFilter):
+    """Filter Resources based on essential contacts configuration, org is optional
+
+    .. code-block:: yaml
+
+      - name: org-essential-contacts
+        resource: gcp.organization
+        filters:
+        - type: essential-contacts
+          org: 99999999
+          category: "ALL"
+    """
+    schema = type_schema('essential-contacts',
+                        org={'type': 'integer'}, category={'type': 'string'})
+    required_keys = {}
+    permissions = ("essentialcontacts.contacts.list",)
+    annotation_key = 'c7n:matched-findings'
+
+    def process(self, resources, event=None):
+        self.findings_list = self.get_findings()
```

Comment:
curious as I saw this in the access filter as well, but why is this called get_findings in both?

Reply:
Thx, Named it more appropriately based on what its doing.